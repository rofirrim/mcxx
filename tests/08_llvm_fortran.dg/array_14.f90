! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_14.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    LOGICAL :: A(4), B(4), C(4)

    A(1) = .TRUE. ; B(1) = .TRUE.
    A(2) = .TRUE. ; B(2) = .FALSE.
    A(3) = .FALSE. ; B(3) = .TRUE.
    A(4) = .FALSE. ; B(4) = .FALSE.

    PRINT *, A
    PRINT *, B

    C = A .AND. B
    PRINT *, C

    C = A .OR. B
    PRINT *, C
END PROGRAM MAIN
