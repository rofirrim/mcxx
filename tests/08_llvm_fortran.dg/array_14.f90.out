 T T F F
 T F T F
 T F F F
 T T T F
